permutations.r

```diff
@@ -1,22 +1,49 @@
-Packages <- c("gtools", "readr", "tibble", "dplyr")
 
-lapply(Packages, library, character.only = TRUE)
+################################################################################
+
+# WHAT IS THIS SCRIPT FOR?
+
+# OH GOD WHAT IS ANYTHING FOR?
+
+################################################################################
+
+packages <- c("gtools", "readr", "tibble", "dplyr")
+
+lapply(packages, library, character.only = TRUE)
+
+
 
 x <- c("A","B","C","D","E","F")
-df <- permutations(n=6, r=6,v=x)
-write.csv(df,"C:\\Users\\xnystl\\Documents\\R projects\\MC-factory\\permutations.csv", row.names = FALSE)
+
+df <- permutations(n = 6, r = 6, v = x)
+
+write.csv(df, "permutations.csv", row.names = FALSE)
```

Comment:
If you're working in an R Project, you won't need the rest of the file path, and indeed, the code will be more robust (i.e., will work on other computers) if you don't use a whole file path.

If there are internal folders in the R Project directory, you might need file path stuff, but the paths start in the R Project directory, rather than at your hard drive.